locallibrary/settings.py

```diff
@@ -16,23 +16,28 @@
 BASE_DIR = Path(__file__).resolve().parent.parent
 
 
+# Add support for env variables from file if defined
+import os
+from dotenv import load_dotenv
+env_path = load_dotenv(os.path.join(BASE_DIR, '.env'))
+load_dotenv(env_path)
+
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
 # SECRET_KEY = 'django-insecure-&psk#na5l=p3q8_a+-$4w1f^lt3lx1c@d*p4x$ymm_rn7pwb87'
```

Comment:
It would be nice to get this out of the `locallibrary/settings.py` file if we're adding dotenv support to make beginners aware that credentials should never be exposed like this even if it's for demos.

Reply:
This is what comes in the Django skeleton application that is built when you start and the key needs to be present. In the instructions we explain you comment this.  So it is useful to show what the user needs to remove/comment out in that application "by comparison". 

I find this useful as a "by comparison" - here's what you had, and here's what you now have.
It also helps me when updating the docs for the new version of Django because I compare and this is a match-ish point.

So yes, but I'm still not going to do it for what I think are better reasons than removing it.